homeassistant/components/cert_expiry/coordinator.py

```diff
@@ -40,6 +45,11 @@ async def _async_update_data(self) -> datetime | None:
         """Fetch certificate."""
         try:
             timestamp = await get_cert_expiry_timestamp(self.hass, self.host, self.port)
+            await validate_cert_expiry(timestamp)
+            self.is_cert_valid = True
```

Comment:
If I understand correctly, this now sets `is_cert_valid` even if the certificate isn't trusted (because no certificate validation other than the hand-coded expiry time check is performed anymore). I think that might be worth a changelog entry at least, since `is_cert_valid` now exclusively represents the timestamp, and nothing else that could be considered relevant for validity.

Reply:
That's true, traditional certificate validation is no longer performed. I'll add that snippet to the PR description.